internal/scheduled/step_remind_certificate_provider_to_complete_test.go

```diff
@@ -20,44 +21,262 @@ import (
 )
 
 func TestRunnerRemindCertificateProviderToComplete(t *testing.T) {
+	donorUID := actoruid.New()
+	correspondentUID := actoruid.New()
+
+	nilNotifyClient := func(*testing.T, context.Context, *lpadata.Lpa) *mockNotifyClient { return nil }
+	nilEventClient := func(*testing.T, context.Context, *lpadata.Lpa) *mockEventClient { return nil }
+
 	testcases := map[string]struct {
+		lpa                      *lpadata.Lpa
 		certificateProvider      *certificateproviderdata.Provided
 		certificateProviderError error
+		notifyClient             func(*testing.T, context.Context, *lpadata.Lpa) *mockNotifyClient
+		eventClient              func(*testing.T, context.Context, *lpadata.Lpa) *mockEventClient
 	}{
-		"certificate provider not started": {
+		"online donor online certificate provider not started": {
+			lpa: &lpadata.Lpa{
+				LpaUID: "lpa-uid",
+				Type:   lpadata.LpaTypePersonalWelfare,
+				Donor: lpadata.Donor{
+					FirstNames:                "a",
+					LastName:                  "b",
+					ContactLanguagePreference: localize.En,
+				},
+				CertificateProvider: lpadata.CertificateProvider{
+					FirstNames:                "c",
+					LastName:                  "d",
+					ContactLanguagePreference: localize.En,
+					SignedAt:                  pt(testNow.AddDate(0, -3, 0).Add(-time.Second)),
```

Comment:
This threw me when I was working through the scenarios. It doesn't affect anything in the test but it's confusing to have a signed date in the LPA store when the CP has not signed - could we drop it?